.gitignore

```diff
@@ -23,6 +23,7 @@ nbproject
 /.php_cs.cache
 /.php-cs-fixer.cache
 /.phpunit.result.cache
+/coverage.xml
```

Comment:
I'd like some of these type of files to be squirrelled away in a subdirectory, though that is beyond the scope of this PR.